capstone_project/src/main/webapp/scripts/comments.js

```diff
@@ -180,3 +233,45 @@ function disableInput(event) { // eslint-disable-line no-unused-vars
   const inputButton = event.target.lastElementChild;
   inputButton.disabled = true;
 }
+
+/**
+ * Increments the count of the chosen reaction locally and on the server side.
+ * @param {HTMLDivElement} element - the requested reaction to send.
+ */
+async function reactToPost(element) { // eslint-disable-line no-unused-vars
+  // the fourth parent is the container for the post, which contains its id.
+  const postID = $(element).parents()[3].dataset.id;
```

Comment:
Is there not a better way to refer to the postID? If it's too difficult right now can you leave a TODO?

Reply:
I found and implemented another way to do it without having to rely on the location of the parent.